src/v2/openapi.yaml

```diff
@@ -0,0 +1,667 @@
+        200:
+          description: Stack report
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StackRecommendationResult'
+        400:
+          description: Bad request
+          content: {}
+
+  /stack_aggregator:
+    post:
+      tags:
+      - Scan Services
+      summary: |
+        Aggregates vulnerabilities, licenses and github statistics associate with list of given packages and writes it into RDS.
+      parameters:
+      - name: persist
+        in: query
+        schema:
+          type: boolean
+          default: true
+        description: |
+          Whether to persist generated report into RDS.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StackAggregatorRequest'
+        required: true
+      responses:
+        200:
+          description: Stack report
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - $ref: '#/components/schemas/StackAggregatorResultForRegisteredUser'
+                - $ref: '#/components/schemas/StackAggregatorResultForFreeTier'
+        400:
+          description: Bad request
+          content: {}
+
+components:
+  schemas:
+    RecommenderRequest:
+      title: RecommenderRequest
+      allOf:
+        - $ref: '#/components/schemas/StackAggregatorRequest'
+      description: EPV details for stack report generation
+
+    StackAggregatorRequest:
+      title: StackAggregatorRequest
+      type: object
+      required:
+        - registration_status
+        - uuid
+        - external_request_id
+        - ecosystem
+        - packages
+        - manifest_file
+        - manifest_file_path
+      properties:
+        registration_status:
+          $ref: '#/components/schemas/RegistrationStatus'
+        uuid:
+          type: string
+          format: uuid
+        external_request_id:
+          type: string
+          example: 229e7da76708fe374d8c10fa752e72989f
+        show_transitive:
+          type: boolean
+          default: true
+          description: |
+            This is required to enable or disable the transitive support
+        ecosystem:
+          $ref: '#/components/schemas/Ecosystem'
+        manifest_file:
+          type: string
+        manifest_file_path:
+          type: string
+        packages:
+          type: array
+          items:
+            $ref: '#/components/schemas/Package'
+      description: EPV details for stack report generation
+
+    Ecosystem:
+      title: Ecosystem
+      type: string
+      enum:
+        - maven
+        - pypi
+        - npm
+      description: List of supported package ecosystems
+
+    BasicVulnerabilityFields:
+      title: BasicVulnerabilityFields
+      required: 
+        - cvss
+        - cvss_v3
+        - severity
+        - title
+        - id
+        - url
+      properties:
+        cve_ids:
```

Comment:
Are these cve_ids not replaced by Synk id's  now ?

Reply:
There is a id field in this model to represent vendor(snyk) vulnerability id.